src/firefly_exchange_client/client.py

```diff
@@ -43,6 +45,7 @@ async def init(self, user_onboarding=True):
 
         if user_onboarding:
             self.apis.auth_token = await self.onboard_user()
+            self.dmsApi.auth_token = await self.onboard_user()
```

Comment:
Why are we invoking `await self.onboard_user()` two times over here? can't we do `self.dmsApi.auth_token` = `self.apis.auth_token` and if both the tokens are same then why do we need to store them separately?